bot/seasons/evergreen/space.py

```diff
@@ -0,0 +1,205 @@
+import logging
+import random
+from datetime import datetime
+from typing import Any, Dict, Optional, Union
+from urllib.parse import urlencode
+
+from discord import Embed
+from discord.ext.commands import BadArgument, Cog, Context, Converter, group
+
+from bot.bot import SeasonalBot
+from bot.constants import Tokens
+
+logger = logging.getLogger(__name__)
+
+NASA_BASE_URL = "https://api.nasa.gov"
+NASA_IMAGES_BASE_URL = "https://images-api.nasa.gov"
+NASA_EPIC_BASE_URL = "https://epic.gsfc.nasa.gov"
+
+APOD_DEFAULT_PARAMS = {
+    "api_key": Tokens.nasa
+}
+
+
+class DateConverter(Converter):
+    """Parse `str` into `datetime` or `int` object."""
```

Comment:
These docstrings could use some more information. What are the valid time strings accepted? What does an integer input represent?